conf/messages

```diff
@@ -124,20 +123,27 @@ nisp.main.context.improve.para2 = Once you have reached the full rate of £{0},
-nisp.excluded.overseas.line3=Austria, Belgium, Bulgaria, Croatia, Cyprus, Czech Republic, Denmark, Estonia, Finland, France, Germany, Gibraltar, Greece, Hungary, Iceland, Ireland, Italy, Latvia, Liechtenstein, Lithuania, Luxembourg, Malta, Netherlands, Norway, Poland, Portugal, Romania, Slovakia, Slovenia, Spain, Sweden or Switzerland.
-nisp.excluded.overseas.line4=<a href="https://www.gov.uk/state-pension-if-you-retire-abroad/how-to-claim" rel="external" target="_blank" data-journey-click="checkmystatepension:external:howtoclaimabroad">Contact the IPC (opens in new tab)</a> if you''ve worked or lived in a country not listed above.
-nisp.excluded.mwrre.line1 = You chose to pay a <a href="https://www.gov.uk/reduced-national-insurance-married-women" rel="external" target="_blank" data-journey-click="checkmystatepension:external:mwrre">reduced rate of National Insurance as a married woman (opens in new tab)</a> and your State Pension has been reduced. You may be able to claim a pension benefit based on your husband''s contributions.
-nisp.excluded.mwrre.line2 = As this is a new service, we have yet to include this condition but we are looking to do so.
+nisp.excluded.title = You are unable to use this service
+
+nisp.excluded.overseas.line1 = If you''ve worked in the UK and abroad, you must send an international claim form to the International Pension Centre (IPC).
+nisp.excluded.overseas.line2 = If you''ve only worked, lived or are working abroad, you must claim your State Pension through the relevant authority of the country you''re in if you currently live and have worked in:
+nisp.excluded.overseas.line3 = Austria, Belgium, Bulgaria, Croatia, Cyprus, Czech Republic, Denmark, Estonia, Finland, France, Germany, Gibraltar, Greece, Hungary, Iceland, Ireland, Italy, Latvia, Liechtenstein, Lithuania, Luxembourg, Malta, Netherlands, Norway, Poland, Portugal, Romania, Slovakia, Slovenia, Spain, Sweden or Switzerland.
+nisp.excluded.overseas.line4 = <a href="https://www.gov.uk/state-pension-if-you-retire-abroad/how-to-claim" rel="external" target="_blank" data-journey-click="checkmystatepension:external:howtoclaimabroad">Contact the IPC (opens in new tab)</a> if you''ve worked or lived in a country not listed above.
+
+nisp.excluded.mwrre.line1 = You chose to pay a <a href="https://www.gov.uk/reduced-national-insurance-married-women" rel="external" target="_blank" data-journey-click="checkmystatepension:external:mwrre">reduced rate of National Insurance as a married woman (opens in new tab)</a> and your State Pension has been reduced. You may be able to claim a pension benefit based on your partner''s contributions.
+nisp.excluded.mwrre.line2 = To get a pension statement, you will need to <a href="https://www.gov.uk/future-pension-centre" rel="external" target="_blank" data-journey-click="checkmystatepension:external:mwrre">contact the Future Pension Centre (opens in new tab)</a>.
```

Comment:
We need to have a different ID for each URL.
```checkmystatepension:external:mwrre``` is in use by ```nisp.excluded.mwrre.line1```
This line should be ```checkmystatepension:external:futurepensioncentre``` or something.